src/Context/Settings/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState, useEffect} from 'react';
+
+
+// create a context object.......
+
+export const SettingsContext = React.createContext();
+
+// create a provider component
+function SettingsProvider({ children }){
+    const [ displayCount, setDisplayCount ] = useState(3);
+    const [ showCompleted, setShowCompleted ] = useState(false);
+    const [ sortField, setSortField ] = useState('difficulty');
```

Comment:
I would probably  suggest checking local storage at the top of your function, and if it's there use those values to initilize state instead of using a `useEffect` hook.